cmd/kaniko-docker/main.go

```diff
@@ -132,6 +140,11 @@ func run(c *cli.Context) error {
 		return err
 	}
 
+	var digestFileName = ""
```

Comment:
remove L143-146. Didn't get what this is doing.

Reply:
So we will generate digest output even if we don't expose plugin artifact file?

Reply:
that is fine

Reply:
removed, as discussed